Sources/Request/Request.swift

```diff
@@ -4,16 +4,20 @@ public struct Request {
 
      public let method: String
      public let path: String
+     public let http: String
      public var queries: [String: String] 
      public var body: [String: String] 
      public var cookie: String  
+     public var headers: [String: String]
 
-     public init(method: String, path: String, queries: [String: String] = ["fname": "World"], body: [String: String] = [String(): String()], cookie: String = String()) {
+     public init(method: String, path: String, http: String = String(), queries: [String: String] = ["fname": "World"], body: [String: String] = [String(): String()], cookie: String = String(), headers: [String: String] = [String(): String()]) {
```

Comment:
Does `headers: [String: String] = [String(): String()]` initialize the dictionary with an empty key and value, i.e., `["": ""]`? If so, why not use an empty dictionary, `[:]` (I think that's the syntax), instead?

Reply:
For some reason this implementation resonated with me. Your way requires less key strokes so that makes sense.